server/routes/induction/update/workedBeforeFormValidator.test.ts

```diff
@@ -0,0 +1,35 @@
+import aValidPrisonerSummary from '../../../testsupport/prisonerSummaryTestDataBuilder'
+import validateWorkedBeforeForm from './workedBeforeFormValidator'
+import YesNoValue from '../../../enums/yesNoValue'
+
+describe('workedBeforeFormValidator', () => {
+  const prisonerSummary = aValidPrisonerSummary()
+
+  describe('happy path - validation passes', () => {
+    it(`has worked before is selected`, () => {
+      // Given
+      const expected: Array<Record<string, string>> = []
+
+      // When
+      const actual = validateWorkedBeforeForm({ hasWorkedBefore: YesNoValue.YES }, prisonerSummary)
+
+      // Then
+      expect(actual).toEqual(expected)
+    })
+  })
+
+  describe('happy path - validation does not pass', () => {
+    it(`has worked before is selected`, () => {
```

Comment:
`has worked before is not selected` ?

Reply:
You're too keen! I've only just created this PR and haven't shared it yet! 😂